environment.yml

```diff
@@ -7,7 +7,7 @@ dependencies:
   - python=3.7
   - pip
   - ctapipe=0.8.0
-  - gammapy>=0.17
+  - gammapy~=0.17
```

Comment:
What prevents the use of gammapy 0.18?

Reply:
This will use any gammapy 0.x version larger than 0.17

Reply:
Ah right. I thought its any 0.17.x version